setupTests-integration.js

```diff
@@ -0,0 +1,37 @@
+import {TestUtils, WALLET_CONSTANTS} from "./__tests__/integration/test-utils-integration";
+import {loggers, TxLogger} from "./__tests__/integration/txLogger";
+import * as Path from 'path';
+
+// const {WebSocket} = require("mock-socket");
```

Comment:
This code was left here as a reference because it will be used in future versions of this integration tests. But since it is available from the `setupTests.js`, this comment can be removed.